monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -0,0 +1,56 @@
+
+Pour faire l'expérience de ce pouvoir, vous devrez choisir un portefeuille. L'aspect le plus critique de tout portefeuille réside dans sa crédibilité. Seuls les portefeuilles recommandés par la communauté (et par le site [getmonero.org](https://getmonero.org/fr/)) sont mentionnés dans cet article. Ensuite, vous voudrez choisir un portefeuille compatible avec votre environnement préféré - iPhone, Android ou ordinateur. Enfin, vous devrez choisir un portefeuille qui répond à vos besoins de confidentialité. Cet article vous guidera sur ce point.
+
+Certains portefeuilles fonctionnent en utilisant leur propre copie de la chaîne de blocs de Monero. On appelle également cela faire fonctionner un nœud complet. Cela vous permet de contrôler la rapidité de vos transactions. Cependant, vous pouvez vous appuyer sur un nœud distant sans risquer perdre vos Monero, vous dépendez alors de la disponibilité du nœud distant (vous serez toujours en mesure de recevoir des Monero, mais vous pourriez ne pas pouvoir en envoyer). Un autre atout du fonctionnement de votre propre nœud complet réside dans la fierté de contribuer au réseau Monero. Les nœuds complets sincères rendent Monero plus fort. Apprenez-en plus ici : [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node).
+
+S'appuyer sur des nœuds distant, au contraire, permet (en utilisant se qui est communément appelé un portefeuille léger) de s'affranchir des ressources de stockage nécessaires à la chaîne de bloc Monero. La chaîne de bloc (non élaguée) fait plus de 50Go et continue de grossir. De nombreux portefeuilles s'appuyant sur des nœuds complets distants utilisent également moins de puissance de calcul et de bande passante réseau. Si ces ressources sont limités sur votre appareil, un portefeuille léger pourrait être préférable. Cela sera presque toujours le cas pour un portefeuille fonctionnant sur Smartphone.
+
+Il y a deux types de portefeuilles légers : à protection ou à partage de clef de lecture. Les portefeuilles légers à protection de clef gardent vos clefs de lecture confidentielles, et doivent donc télécharger la chaîne de bloc, ou du moins les portions plus récentes que la date de création du portefeuille, pour pouvoir déterminer le solde du portefeuille. Les portefeuilles légers à partage de clef, par opposition, partage vos clefs de lecture avec le serveur exécutant le nœud distant pour calculer le solde du portefeuille depuis celui-ci. Ces portefeuilles performants sont parfois appelés applications de portefeuilles. Le partage de vos clefs de lecture amoindrit la confidentialité si le serveur exécutant le nœud distant et connaissant les clefs de lecture n'est pas fiable car il permet l'observation des transactions entrantes. Remarquez, cependant, que des propriétaires de portefeuilles partagent parfois leurs clefs de lecture pour des besoins d'audit (elles ne permettent pas de transférer des Monero).
```

Comment:
'pour des besoins d'audit' ---> 'pour des besoins d'audit de toute façon'
Reason: 'Anyway' appears in the original english document.

Reply:
I disagree, we are not supposed to keep every word unchanged (as you mentioned yourself about network bandwidth).  
to me, adding "de toute façon" make the sentence really inelegant.

Reply:
Agreed that this does not add so much value. The sentence is a bit long and adding 'de toute façon' requires to read the sentence fast to assimilate it. So, for the sake of clarity, I don't see any problem in keeping your original text.